modules/addons/grafana/main.tf

```diff
@@ -13,6 +13,10 @@ provider "kubectl" {
   load_config_file       = false
 }
 
+resource "random_password" "grafana" {
```

Comment:
let's use  `"tsb_password"` as a variable to set set password for all the related tools, for example argocd password is using `tsb_password` as well.

Reply:
Ok, here and in the other user-facing things I've changed it a bit to use the `tsb_password` if set, but to generate a random secure password if not set. The `tsb_password` is no longer mandatory, and if omitted a random secure pass will be set for each component. If set, TSB, Grafana and ArgoCD will use the same tsb password.

Reply:
even better! thank you!